lib/ziti-tunnel-cbs/ziti_tunnel_ctrl.c

```diff
@@ -301,7 +343,7 @@ static int load_identity(const char *identifier, const char *path, command_cb cb
 
 static struct ziti_instance_s *new_ziti_instance(const char *identifier, const char *path) {
     struct ziti_instance_s *inst = calloc(1, sizeof(struct ziti_instance_s));
-    inst->identifier = strdup(identifier ? identifier : path);
+    inst->identifier = strdup(identifier ? identifier : extract_filename(path));
```

Comment:
memory leak: extract_filename allocates memory

Reply:
removed extract_filename function as per the review comment in the other PR